### PR TITLE
Gdr 2601

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: GDR-2601

## Why was it changed?
Added support for using custom functions for calculating bliss and hsa scores

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: R/combinations-helpers.R [hunk 1]
```diff
@@ -112,6 +112,42 @@ calculate_excess <- function(metric,
...
+#'   metric_col, 
+#'   measured_col
+#' )
+#' calculate_score(x$x)
+#'
+#' @return numeric vector with calculated score
+#' @keywords combinations
+#' @export
+calculate_score <- function(excess) {
+  checkmate::assert_numeric(excess)
```
Comment (gladkia) [inline R/combinations-helpers.R:146]: consider adding protection against all NAs in this function or in the parent function.
Sth like:
```
nav <- rep(NA_integer_, 100)
if (all(is.na(nav))) {
  stop("Sth wrong - provided input data contains NAs only.")
}
```

**Review by gladkia — APPROVED** [review]

**Review by marcin-kam — APPROVED** [review]

File: R/combinations-helpers.R (hunk 1 above)
Comment (bczech) [reply R/combinations-helpers.R:146]: All NAs will occur in some designs (cotreatment) data and it's fine, we shouldn't break the app and keep NA as we do

Comment (gladkia) [reply R/combinations-helpers.R:146]: I see. Could you consider adding the unit test for such an edge case then?

Comment (bczech) [reply R/combinations-helpers.R:146]: Done :)
